variables.tf

```diff
@@ -197,6 +197,25 @@ variable "dapr_component_secrets" {
   sensitive   = true
 }
 
+variable "env_storage" {
+  type = map(object({
+    name                 = string
+    storage_account_name = string
```

Comment:
Would `var.env_storage.storage_account_name` a bit verbose? Could we rename it to `account_name`?